test/integration/integration_stream_decoder.h

```diff
@@ -37,7 +40,8 @@ class IntegrationStreamDecoder : public Http::ResponseDecoder, public Http::Stre
   // can be used if the previous body data is not relevant and the test wants to wait for a specific
   // amount of new data without considering the existing body size.
   void waitForBodyData(uint64_t size);
-  void waitForEndStream();
+  testing::AssertionResult
```

Comment:
WDYT about making this ABSL_MUST_USE_RESULT? (Otherwise will the AssertionFailure fail unchecked?)

Reply:
yeah, it should be but it breaks the filter tests.
My plan is to turn this up for this and waitforreset, fix the filter tests (and ping import rotation) then make it required.

Reply:
Ohhh.. pains of a separate repo! Thanks.